### PR TITLE
Fix: empty all fields after case creation.

### PR DESCRIPTION
**Pull request by AAJELLAL**

### CODE REVIEW

**Review by Walid-Sahnoun — APPROVED** [review]
review ok
tests ok

File: src/components/dialogs/field-hook.js [hunk 1]
```diff
@@ -137,6 +141,9 @@ export const useFileValue = ({ fileExceedsLimitMessage, isLoading }) => {
             setFileOk(false);
         }
     }, [selectedFile, fileExceedsLimitMessage, intl]);
+    if (noSetFileNeeded) {
```
Comment (SlimaneAmar) [inline src/components/dialogs/field-hook.js]: Don't understand

**Review by SlimaneAmar — CHANGES_REQUESTED** [review]

File: src/components/dialogs/field-hook.js (hunk 1 above)
Comment (AAJELLAL) [reply src/components/dialogs/field-hook.js]: I used it in order to return specific fields. But as discussed there is another way to do it.

**Comment by flomillot** [comment]
Sorry but I'm pretty busy

**Review by SlimaneAmar — APPROVED** [review]

**Comment by SlimaneAmar** [comment]
review ok
tests ok

**Review by Walid-Sahnoun — APPROVED** [review]
code ok 
test ok

**Pull request merged**